charts/argocd-apps/templates/projects.yaml

```
@@ -22,6 +22,7 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
+  permitOnlyProjectScopedClusters: {{ .permitOnlyProjectScopedClusters | default false }}
```

Can you try if this would also work? I'd prefer to follow our code style which is already in place in the lines below.
```suggestion
  {{- with .permitOnlyProjectScopedClusters }}
  permitOnlyProjectScopedClusters: {{ . }}
  {{- end }}
```

---

Sure @mkilchhofer ! let me try

---

```
➜  argocd-apps git:(add_permitonlyscopedprojects) ✗ helm template . -f values2.yaml                                                                                                                                                                                                                   (⎈ |kind-kind:default)
---
# Source: argocd-apps/templates/projects.yaml
apiVersion: argoproj.io/v1alpha1
kind: AppProject
metadata:
  name: orka-extensibility
  namespace: cre-system
spec:
  permitOnlyProjectScopedClusters: true
  description: orka-extensibility
  sourceRepos:
    - '*'
  destinations:
    - namespace: '*'
      server: '*'
  clusterResourceWhitelist:
    - group: '*'
      kind: '*'
  orphanedResources:
    warn: true
  roles:
    - description: Read-only privileges to orka-extensibility
      groups:
      - OrkA Team
      name: read-only
      policies:
      - p, proj:orka-extensibility:read-only, applications, get, orka-extensibility/*,
        allow
      - p, proj:orka-extensibility:read-only, applicationsets, get, orka-extensibility/*,
        allow
      - p, proj:orka-extensibility:read-only, applications, sync, orka-extensibility/*,
        allow
      - p, proj:orka-extensibility:read-only, applicationsets, sync, orka-extensibility/*,
```